hubgrep/models/__init__.py

```diff
@@ -64,16 +65,18 @@ def set_service_label(self):
         self.label = re.split("//", self.landingpage_url)[1].rstrip("/")
 
     def get_hosting_service_interface(self, cached_session: 'CachedSession', timeout: int) -> "HostingServiceInterface":
-        config_str = self.config
-        config = json.loads(config_str)
+        # todo: add this, when we know what to do with it :P
+        #custom_config_str = self.custom_config
+        #custom_config = json.loads(custom_config_str)
```

Comment:
the custom config is unused right now.
(we cant assume that we just want to set it as additional headers, or can we?)

Reply:
I think so, but to go with this solution, the safest is to put a "headers" key in the config

Reply:
so... we expect a dict that has something like `additional_headers`?
we could make just a db field for that if we need it, then we wouldnt have to unpack an arbitrary config dict 
:clown_face: